code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +167,39 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone
+	name = "saxophone"
+	desc = "NEVER GONNA DANCE AGAIN, GUILTY FEET HAVE GOT NO RHYTHM!"
+	icon = 'icons/obj/musician.dmi'
+	icon_state = "sax"
+	item_state = "sax"
+	force = 10
+	cooldowntime = 150
+	attack_verb = list("played blues on", "serenaded", "battered", "bashed")
+	honksound = list('sound/items/sax.ogg', 'sound/items/sax2.ogg','sound/items/sax3.ogg','sound/items/sax4.ogg','sound/items/sax5.ogg','sound/items/sax6.ogg')
+
+
+/obj/item/weapon/bikehorn/saxophone/attack_self(mob/user)
+	if(!spam_flag)
+		spam_flag = TRUE
+		playsound(src.loc, pick(honksound), 50, 0)
+		user.visible_message("<B>[user]</B> lays down a [pick("sexy", "sensuous", "libidinous","spicy","flirtatious","salacious","sizzling","carnal","hedonistic")] riff on \his [src]!")
+		add_fingerprint(user)
+		spawn(cooldowntime)

```

Comment:
Please use the `addtimer` and `callback` system to do this instead of using spawns, you can find more information about this [here](https://github.com/HippieStation/HippieStation13/blob/master/code/datums/callback.dm)